chart/kubeapps/templates/NOTES.txt

```diff
@@ -8,6 +8,14 @@ Kubeapps can be accessed via port {{ .Values.frontend.service.port }} on the fol
 
    {{ template "kubeapps.fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local
 
+{{- range .Values.apprepository.initialRepos }}
```

Comment:
we can live with this, but one thing you could try: move this to a template and call the template using `include` (https://github.com/helm/helm/blob/master/docs/charts_tips_and_tricks.md#using-the-include-function) to get a string back, check for the string to not be empty and print if not empty.

Reply:
I tried with `template` but not with `include`, that did the trick!

Reply:
nice 🎉!!!